ts/de_DE.ts

```diff
@@ -52,7 +52,7 @@
     <message>
         <location filename="controller.py" line="1837"/>
         <source>Exporting G-code to</source>
-        <translation>Exportiere GCode nach</translation>
+        <translation>Exportiere gcode nach</translation>
```

Comment:
Yes agreed consistency is a big point to remove user confusion through different spellings.